src/main/java/org/openstreetmap/atlas/geography/converters/WkMultiPolygonConverter.java

```diff
@@ -0,0 +1,56 @@
+package org.openstreetmap.atlas.geography.converters;
+
+import org.locationtech.jts.geom.Geometry;
+import org.openstreetmap.atlas.exception.CoreException;
+import org.openstreetmap.atlas.geography.MultiPolygon;
+import org.openstreetmap.atlas.geography.converters.jts.JtsMultiPolygonToMultiPolygonConverter;
+import org.openstreetmap.atlas.geography.converters.jts.JtsPolygonToMultiPolygonConverter;
+import org.openstreetmap.atlas.utilities.conversion.TwoWayConverter;
+
+/**
+ * @author matthieun
+ * @param <T>
+ *            The type to convert to: {@link String} or byte[]
+ */
+public abstract class WkMultiPolygonConverter<T> implements TwoWayConverter<MultiPolygon, T>
```

Comment:
Nice refactor! The names had me confused for a sec :)

Reply:
Wait what is the significance of the `Wk`?

Edit: Oh never mind I understand now. This is a superclass for `WKT` or `WKB`. Maybe add that to the javadoc? :)